websites/E/EWAIFU/presence.ts

```diff
@@ -0,0 +1,68 @@
+const presence = new Presence({
+	clientId: "1224473731070689331",
+});
+
+const enum Assets {
+	Logo = "https://www.ewaifu.gg/img/logo.png",
+	Home = "https://www.ewaifu.gg/img/home.png",
+}
+
+const timestampCheck: {
+		hash: string;
+		timestamp: number;
+	} = {
+		hash: "",
+		timestamp: Math.floor(Date.now() / 1000),
+	},
+	staticPages: Record<string, PresenceData> = {
+		"/": {
+			details: "Discovering Ewaifus",
+			smallImageKey: Assets.Home,
+			buttons: [{ label: "Discover Ewaifus", url: "https://www.ewaifu.gg/" }],
```

Comment:
Buttons redirecting to main pages are not allowed by the [guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets)
```suggestion
```